### PR TITLE
Dnaiel0809 fix rich list

### PR DESCRIPTION
**Pull request by dnaiel0809**
Fix issue#1
Merge the GraphQL queries into one.
The rich list will be "Loading" while waiting for the fetching.
Add my contribution md.
Now the average time for query Blockchairapi is below 3s.

### CODE REVIEW

File: contributors/yujen-lee.md
Comment (erinmhpark) [inline contributors/yujen-lee.md:1]: Name of the contributor is incorrect. Please update the name in the file to your own.

File: tests/home.spec.ts
Comment (erinmhpark) [inline tests/home.spec.ts:1]: Please do not change the test file for other's component.

File: tests/min.spec.ts
Comment (erinmhpark) [inline tests/min.spec.ts:1]: Please do not change the test file for other's component.

**Review by erinmhpark — COMMENTED** [review]
@dnaiel0809 
Please refer to the comments I made on this PR. 
I can't approve this PR. It needs to be updated: 
1. Update the name of contributor 
2. Do not changes other's test files

**Comment by dnaiel0809** [comment]
Hi, @erinmhpark I fixed the issues that you left in the comment. Please kindly review them. Thanks.

**Review by erinmhpark — APPROVED** [review]
Commented issues resolved.

**Comment by dnaiel0809** [comment]
solved the page.tsx conflict.

**Review by erinmhpark — APPROVED** [review]
Commented issues resolved.

**Pull request closed without merging**